### DISCUSSION TITLE
Add NextJS as a curriculum.

### DISCUSSION THREAD

**Discussion by rupokghosh** [TheOdinProject/curriculum#27933] [Ideas]
With the increasing popularity and use of NextJS, I think it's an absolute necessity for it to be added to the TOP curriculum. In the modern world, NextJS is seemingly becoming more and more popular with not only enthusiasts but also companies, and therefore I believe, MERN alone is not enough to become a full-stack developer.

**Comment by mao-sz** [comment]
Hi @rupokghosh. While we may consider mentioning further frameworks as a whole (so not NextJS exclusively but things like it) as a part of the Node course revamp that's a work in process, we are unlikely to add NextJS itself as an actual taught part of the curriculum.

Fundamentals are more important than experience with specific technologies at the entry level, and experience with NextJS is not an actual requirement of many job positions (this is even if the job listing for an entry-level position mentions that the team uses NextJS).

What TOP is focused on is *not* teaching everything. Instead, it's focused on teaching fundamentals that can get someone to a position where learning things like NextJS (or any other framework) is fine based on their fundamental knowledge, and ability to read documentation and research. That is a significantly more valuable skill in the workplace that employers will be looking for. Especially since many jobs won't necessarily even be using NextJS but something else. Fundamentals will take you wherever you want to go.

TOP would spend its time much better by limiting what it teaches to the those fundamentals, and trying its best to do them well. Hope that makes sense regarding the goal and philosophy of this curriculum.
